src/assets/stylesheets/index.css

```diff
@@ -0,0 +1,173 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+html {
+  -webkit-tap-highlight-color: transparent;
+  box-sizing: border-box;
+  font-size: 20px;
+  line-height: 1.5;
+}
+
+body {
+  background: #fff;
+  color: #3b4351;
+  font-family: -apple-system,system-ui,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,sans-serif;
+  font-size: .8rem;
+  overflow-x: hidden;
+  text-rendering: optimizeLegibility;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+figure,
+blockquote,
+dl,
+dd {
+  margin: 0;
+}
+
+a {
+  color: inherit;
+  -webkit-tap-highlight-color: transparent;
+}
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+.h3, h3 {
+  font-size: 1.4rem;
+}
+h1, h2, h3, h4, h5, h6 {
+  color: inherit;
```

Comment:
There are problems with formatting, add stylelint linter as in bsa-website repository